pages/common/srm.md

```diff
@@ -1,17 +1,19 @@
 # srm
 
 > Securely remove files or directories.
-> Overwrites the existing data one or multiple. Drop in replacement for rm.
+> Overwrites the existing data one or multiple times. Drop in replacement for rm.
 
-- Remove a file after overwriting (single pass, 7 pass, 35 pass):
+- Remove a file after a single pass overwriting with random data:
```

Comment:
I'd suggest either "a single-pass overwriting" or "a single pass of overwriting"